fuels-abigen-macro/Cargo.toml

```diff
@@ -21,7 +21,7 @@ syn = "1.0.12"
 
 [dev-dependencies]
 fuel-core = { version = "0.2", default-features = false }
-fuel-gql-client = { version = "0.2", default-features = false }
+fuel-gql-client = { version = "0.3", default-features = false }
```

Comment:
You probably want to bump `fuel-core` in line with `fuel-gql-client`, since they're versioned in lockstep.

Reply:
:facepalm: it's obvious, `gql-client` is litteraly part of `fuel-core`. And that fixes the tests, thanks a lot!